### ISSUE TITLE
Language of parts not indicated

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4579]
When another language is picked for the website, parts are translated. The entire page is indicated as having switched to the new language. (There is a lang-attribute on the html-element)
However, as not everything is in the selected language, some parts should be marked as being a different language.

The corresponding WCAG criterion can be found here: https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos
